crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -2787,7 +2789,7 @@ async fn test_authority_persist() {
         AuthorityStore::open_with_committee_for_testing(&path, None, &committee, &genesis, 0)
             .await
             .unwrap();
-    let authority = init_state(committee, authority_key, store).await;
+    let authority = init_state(&genesis, authority_key, store).await;
```

Comment:
Why do we call `init_state` here but not on line 2817 when we initialize `authority2`?